backend/db/data/movies/fb_1.json

```diff
@@ -18,6 +18,7 @@
   "budget": "$175–200 million",
   "box_office": "$814 million",
   "rating": "PG-13",
+  "trailer":"https://www.youtube.com/watch?v=VYZ3U1inHA4",

```

Comment:
For all of those trailers, could you please use the trailers from one single yt channel? You can use for example [this one](https://www.youtube.com/wbpictures/search?query=harry%20potter%20final%20trailer).